runtime/drivers/sqlite/registry.go

```diff
@@ -63,15 +63,17 @@ func (c *connection) CreateInstance(_ context.Context, inst *drivers.Instance) e
 	now := time.Now()
 	_, err := c.db.ExecContext(
 		ctx,
-		"INSERT INTO instances(id, olap_driver, olap_dsn, repo_driver, repo_dsn, embed_catalog, created_on, updated_on) "+
-			"VALUES ($1, $2, $3, $4, $5, $6, $7, $7)",
+		"INSERT INTO instances(id, olap_driver, olap_dsn, repo_driver, repo_dsn, embed_catalog, created_on, updated_on, env, project_env) "+
+			"VALUES ($1, $2, $3, $4, $5, $6, $7, $7, $8, $9)",
 		inst.ID,
 		inst.OLAPDriver,
 		inst.OLAPDSN,
 		inst.RepoDriver,
 		inst.RepoDSN,
 		inst.EmbedCatalog,
 		now,
+		inst.Env,
+		inst.ProjectEnv,
```

Comment:
Would prefer explicit JSON serialization/deserialization in `findInstances` and `CreateInstance` here. Some reasons:
- Other drivers might have better type support (like map or JSON type support), so the `driver.Valuer` and `sql.Scanner` implementation on `Env` might interfere
- Makes it easier to understand (more explicit) when reading this code that it gets stored as JSON

Reply:
Its a better idea to implement the json serialisation and deserialisation as part of `driver.Valuer` and `sql.Scanner` rather than while generating sql queries. 

- There could be multiple such queries in future like update, different select etc and we would need to copy this logic everywhere. Its better to handle such translations at single place. Easy to remove or modify them if required.
- Agreed on drivers supporting map or JSON support. But in that case I think we can completely remove the use of `driver.Valuer` and `sql.Scanner` impls. 
- Not sure on JSON readability. Its looking on the `driver.Valuer` and `sql.Scanner` vs looking at the query generation code to identify that data is stored as json. May be we can add comments at the struct definition as well.

Reply:
Fair enough about wanting to handle serialization one place. Then can we move it into the `sqlite` driver and use `map[string]string` here (since the JSON serialization is specific to the `sqlite` driver)?

For context, we'll be adding a Postgres driver in addition to the SQLite driver quite soon (for hosted runtimes). So this would come in the way then (Postgres has `JSONB` which would take care of this).

Reply:
I can picture two approaches to add a direct driver level support for map[string]string:
- Make a change in the sqlite driver itself
- Write a wrapper over scan and query methods of sqlite driver
None of these two seems good.

IMO we need to think (as part of a bigger question) on how we are going to handle same struct definitions for both sqlite and Postgres since Postgres's type support is expected to be a lot richer than that of sqlite. This may not be just limited to `JSONB`.